include/HubFramework/HUBViewControllerScrollHandler.h

```diff
@@ -78,6 +78,17 @@
 - (UIEdgeInsets)contentInsetsForViewController:(HUBViewController *)viewController
                          proposedContentInsets:(UIEdgeInsets)proposedContentInsets;
 
+
+/**
+ *  Return the center point of an overlay coponent used in a view controller
+ *
+ *  @param viewController The view controller in question
+ *  @param proposedCenterPoint The center point that the Hub Framework is proposing
+ *
+ */
+- (CGPoint)centerPointForOverlayComponentInViewController:(HUBViewController *)viewController
```

Comment:
I totally agree with @Lumenela, I don't see how this has anything to do with scrolling. We want to stick to the single responsibility principle as much as possible.